aten/src/ATen/native/Cross.cpp

```diff
@@ -1,52 +1,59 @@
 #include <ATen/ATen.h>
 #include <ATen/Dispatch.h>
 #include <ATen/NativeFunctions.h>
+#include <ATen/native/Resize.h>
+#include <ATen/ExpandUtils.h>
 
 #include <ATen/native/Cross.h>
 
 namespace at { namespace native {
 
 DEFINE_DISPATCH(cross_stub);
 
+int64_t _default_cross_dim(const c10::optional<int64_t> &dimension, IntArrayRef sizes) {
+  if(dimension.has_value()) {
```

Comment:
Add a comment explaining what this function does + how it's intended to be used

Reply:
Definitely, nice idea, I'll do that.